packages/angular/projects/ui-angular/src/lib/primitives-poc/flex/flex-component/flex.component.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  Renderer2,
+} from '@angular/core';
+import { AmplifyBasePrimitiveComponent } from '../../base-primitive';
+
+const $COMPONENT_SELECTOR: string = 'amplify-flex';
+
+@Component({
+  selector: $COMPONENT_SELECTOR,
+  templateUrl: 'flex.component.html',
+})
+export class AmplifyFlexComponent {
+  @Input() wrap: string;
```

Comment:
_Future improvement_, maybe we can strongly type 'string' into specific values. Maybe there will be a html types library we can reference?

Reply:
Yes